test/tests/functional/pbs_soft_walltime.py

```diff
@@ -880,9 +880,12 @@ def test_soft_held(self):
 
         # Wait for some more time and verify that soft_walltime
         # extending again
-        time.sleep(7)
+        self.logger.info(
+            "Sleep to let soft_walltime get extended again")
```

Comment:
since we have learnt that why 17s is needed could you make the log message bit more explicit